araproc/analysis/correlation_snr.py

```diff
@@ -0,0 +1,254 @@
+# this calculates correlation snr of pair of channels (ch) separately 
+# for VPols (RF ch 0-7) and HPols (RF ch 8-15)
+
+import scipy
+import itertools
+import numpy as np
+from scipy.signal import correlate as co
+
+from araproc.analysis import snr
+from araproc.framework import waveform_utilities as wfu
+
+
+def channel_pairs():
```

Comment:
I think to keep things simple, we'll want to put all of your correlation_snr.py into standard_reco.py. A lot of these functions already exist there and are precalculated.

Reply:
Does that mean, we are not keeping a dedicated correlation_snr.py script?

Reply:
Right, I think in this case we don't need it.